nix/packages.nix

```diff
@@ -295,6 +288,12 @@ in rec {
   }
   else {});
 
+  # On CI we only cache a subset of cached since we only build a subset of targets.
```

Comment:
We also use CI to populate the Nix cache to benefit developers on their machines. This comment makes it sound as if CI should only build Nix derivations that are strictly required by CI itself.

Are we still pushing `texlive` to Nix cache somewhere, or will macOS users that want it have to build it on their machines?

Reply:
we don’t patch `texlive` so you will get it from the upstream cache or rather from the original host since the tex nix derivations are marked as uncached (or whatever this attribute is called) since it’s mainly fetching a ton of small binary packages.